docs/release/release_series.md

```diff
@@ -19,12 +19,25 @@ will diverge over time.
 Our goal is, within a series, that one may upgrade their OSG services via `yum update` cleanly and without any necessary
 config file changes or excessive downtime.
 
-OSG Release Series
-------------------
+Series Overviews
+----------------
 
 Since the start of the RPM-based OSG software stack, we have offered the following release series:
 
--   **OSG 3.5** started August 2019.
+!!! danger "Before considering an upgrade to OSG 3.6&hellip;"
+    Due to potentially disruptive changes in protocols, contact your VO(s) to verify that they support token-based
+    authentication and/or HTTP-based data transfer before considering an upgrade to OSG 3.6.
+    If you do not know which VOs you are currently supporting, contact us at <help@opensciencegrid.org>.
+
+-   **OSG 3.6** (started February 2021) overhauls the authentication and data transfer protocols used in the OSG
+    software stack:
+    [bearer tokens](https://scitokens.org/) are used for authentication instead of GSI proxies and HTTP is used for data
```

Comment:
```suggestion
    [SciTokens](https://scitokens.org/) are used for authentication instead of GSI proxies and HTTP is used for data
```
or something. At least mention the word SciTokens somewhere so you can search for it.

Reply:
I think this change would be up to @bbockelm since only referring to SciTokens here is likely to cause confusion for US LHC sites since LHC VOs will be using WLCG tokens

Reply:
"bearer tokens such as [SciTokens](https://scitokens.org) and WLCG tokens"?

Reply:
That sounds good to me